Src/Base/AMReX_RealVect.H

```diff
@@ -521,13 +521,13 @@ public:
   /**
      This is a RealVect all of whose components are equal to zero.
   */
-  static AMREX_EXPORT const RealVect Zero;
```

Comment:
Why is `AMREX_EXPORT` removed? I think we need it for Windows. @ax3l

Reply:
Fixed.